readme.md

```diff
@@ -1,51 +1,153 @@
-# Ewon Connector
+
+# Ignition Ewon Connector
+
+## [Table of Contents](#table-of-contents)
+
+1. [Description](#description)
+2. [Version](#version)
+3. [Support](#support)
+4. [Features](#features)
+5. [Download](#download)
+6. [Installation](#installation)
+7. [Setup](#setup)
+8. [Connector Usage](#connector-usage)
+
+
+## [Description](#table-of-contents)
 
 Synchronize Ewon data to Ignition's Tag Historian
 
-## Version
+## [Version](#table-of-contents)
 
 This module version is compatible with Ignition 8.  For an Ignition 7.9 compatible module go to [Ignition-7.9-master](https://github.com/hms-networks/eWonConnector/tree/Ignition-7.9-master) branch.
 
-## Download
+## [Support](#table-of-contents)
 
-The module can be downloaded from the [releases page](https://github.com/hms-networks/eWonConnector/releases).
+This application is supported by HMS' North American offices.
+
+| Phone                                        | Forum                                      | Email                                           |
+|:--------------------------------------------:|:------------------------------------------:|:-----------------------------------------------:|
+| [+1 312 829 0601](tel:13128290601), Option 2 | [hms.how](https://forum.hms-networks.com/) | [support@hms.how](mailto:support@hms.how)       |
 
-## Features
+## [Features](#table-of-contents)
 
-* Data synchronization between Ewon DataMail and Ignition
-   * Tag value updated up to once per minute
+* Data synchronization between Ewon DataMailbox and Ignition
+   * Tag values updated up to once per minute
 * Realtime Read functionality
    * Faster tag value update rates via the Talk2M API
-   * Supports up to once per second update rates
-   * Configured on a tag by tag basis by setting a tag's `Realtime` property to true
-   * Configured on a per Ewon basis by setting the `AllRealtime` tag in an Ewon's `_config` directory to true.
+   * Tag values updated up to once per second
 * Write functionality
    * Allows Ignition to modify tag values on a Flexy
 
-## Installation
+## [Download](#table-of-contents)
+
+The module can be downloaded from the [releases page](https://github.com/hms-networks/eWonConnector/releases).
+
+## [Installation](#table-of-contents)
 
-The module can be installed by navigating to the Ignition web server.  Once on the webserver go to the "Config" section then under the "System" category click the "Modules" page.  At the bottom of the "Modules" page click "Install or Upgrade a Module...".  Follow the prompts to install the module.
+1.  On the Gateway Webpage, select `Config` > `SYSTEM` > `Modules` to open the Module Configuration page.
 
-## Configuration
+      <img src="https://raw.githubusercontent.com/hms-networks/IgnitionEwonConnector/Ignition-8-master/images/module_config_page.JPG" alt="Ignition Module Page" width="1000"/>
```

Comment:
The images should probably not use the full GitHub raw link. They can be put into the readme with relative paths.

The syntax is:
![Ignition Module Page](./images/module_config_page.JPG "Module Config Page"

The readme for the automation fair demo uses the relative path syntax: https://github.com/hms-networks/automationFairDemo/blob/master/README.md

Reply:
Raw link is useful for local copies of the readme

Reply:
What do you use for viewing the readme locally? I've used the relative links locally before, and its worked.

Reply:
releases generally do not include readme image resources as they clutter up the release directory

Reply:
oh, ok. makes sense